packages/app-mobile/components/screens/Note.tsx

```diff
@@ -434,8 +530,72 @@ class NoteScreenComponent extends BaseScreenComponent {
 		// has already been granted, it doesn't slow down opening the note. If it hasn't
 		// been granted, the popup will open anyway.
 		void this.requestGeoLocationPermissions();
+
+		this.unsubscribeScreenWidthChangeHandler_ = Dimensions.addEventListener('change', this.handleScreenWidthChange_);
 	}
 
+	private animateNotesBarOpen = () => {
```

Comment:
I think the animation will still run when the component is unmounted - can that be an issue? Please check.

Also 1.5 seconds to close or open seems too slow, but I'll need to check on device.

Reply:
The animation doesn't run automatically, it has to be triggered by pressing the toggle button. There'll be no way to click the button when the component is unmounted, so I don't think it'll run when the component unmounts.

Reply:
Well it will run if the animation starts then, while it's running, the component is unmounted.

Reply:
Oh, I'll check

Reply:
So, I checked and the animation runs after unmounting, I don't think it'll have any impact on performance though, since it will run only once.

Also, I discovered a type error after importing react native components using the import syntax, instead of require(), Animated.Spring doesn't have a duration option, so it's been removed now.